Spanner/src/Batch/BatchClient.php

```diff
@@ -217,7 +215,7 @@ public function snapshotFromString($identifier)
 
         return $this->operation->createSnapshot($session, [
             'id' => $data['transactionId'],
-            'readTimestamp' => $this->createTimestampWithNanos($data['readTimestamp'], Timestamp::class)
+            'readTimestamp' => Timestamp::createFromString($data['readTimestamp'], Timestamp::PRECISION_NANOSECOND)
```

Comment:
Maybe I'm reading this wrong, but doesn't this only take 1 argument? (https://github.com/GoogleCloudPlatform/google-cloud-php/pull/1002/files#diff-723dc469e6c5b62be473e117ac52e1beR96)

Reply:
thanks -- it's an artifact from a earlier iteration which I forgot to remove. forthcoming update will fix all references.